### PR TITLE
Add issue and pull request template and links to the astropy contributing guidelines

### PR DESCRIPTION
**Pull request by MSeifert04**
The issue and pr template are explained in https://github.com/blog/2111-issue-and-pull-request-templates.

The contribution guideline is explained here: https://github.com/blog/1184-contributing-guidelines

Text is provisional!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+This is the template for bug reports, if you have a feature request you can
+safely ignore and delete this prefilled text.
+
+Include a description of the problem: What are you trying to do? What did you
+expect?
+
```
Comment (crawfordsm) [inline .github/ISSUE_TEMPLATE.md]: If possible, please also include how you were running ccdproc and any error message that was produced

**Comment by crawfordsm** [comment]
Thanks for adding this -- it looks like a great start and I liked how you phrased API changes without using API changes.   Looks like a good first pass and I added a few comments!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Please have a look at the following list and replace the "[ ]" with a "[x]" if
+the answer to this question is yes.
+
+- [ ] For new contributors: Did you add yourself to the "Authors.rst" file?
+- [ ] For bugfixes and new functionality: Did you add an entry to the
+      "Changes.rst" file?
+- [ ] For bugfixes: Did you add a regression test?
```
Comment (crawfordsm) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can these be grouped together as in:

For bugfix pull requests:
   item1
   item2
   ...

For new functionality:

**Comment by MSeifert04** [comment]
@crawfordsm Thank you! I think I've adressed your comments.

**Comment by bsipocz** [comment]
Would you consider changing the order of groups for the PR checkboxes? First bug fixes and features, and then the docs skip ci and adding the contributor, as those are less frequent?

**Comment by bsipocz** [comment]
Otherwise it's great to have a template, especially for bug reporting.

**Pull request merged**